src/pages/Home.vue

```diff
@@ -1,6 +1,122 @@
 <template>
   <main>
-    <h1>LevelDB Inspector</h1>
-    <img src="@images/logo.png" alt="App Logo" />
+    <header>
+      <div class="logo"></div>
+    </header>
+    <form>
+      <p>Select path to LevelDB</p>
+      <label class="file db-path">
+        <input type="file" id="file" aria-label="File browser example" />
+        <span class="file-custom"></span>
+      </label>
+      <input type="submit" value="Inspect" />
+    </form>
+    <footer>Copyright 2020 • UncaughtException</footer>
   </main>
 </template>
+
+<style scoped>
+main {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.logo {
+  width: 150px;
+  height: 150px;
+  background-color: #f7f7f7;
+  border-radius: 100%;
+  margin: 100px auto;
+}
+
+form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.file {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  height: 2.5em;
+}
+
+.file input {
+  min-width: 35em;
+  margin: 0;
+  filter: alpha(opacity=0);
+  opacity: 0;
+}
+
+.file-custom {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 5;
+  height: 2.5em;
+  padding: 0.5em 1em;
+  line-height: 1.5;
+  color: #555;
+  background-color: #fff;
+  border: 0.075em solid #ddd;
+  border-radius: 0.25em;
+  box-shadow: inset 0 0.2em 0.4em rgba(0, 0, 0, 0.05);
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
@jjlumagbas This will be an Electron app, which is based on Chrome, so we don't need the other browser prefixes anymore.